Gemfile.lock

```diff
@@ -1,21 +1,93 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    async (2.3.1)
+    async (2.4.2)
       console (~> 1.10)
       io-event (~> 1.1)
       timers (~> 4.1)
+    async-container (0.16.12)
+      async
+      async-io
+    async-http (0.60.1)
+      async (>= 1.25)
+      async-io (>= 1.28)
+      async-pool (>= 0.2)
+      protocol-http (~> 0.24.0)
+      protocol-http1 (~> 0.15.0)
+      protocol-http2 (~> 0.15.0)
+      traces (>= 0.8.0)
+    async-http-cache (0.4.3)
+      async-http (~> 0.56)
+    async-io (1.34.3)
+      async
+    async-pool (0.4.0)
+      async (>= 1.25)
+    build-environment (1.13.0)
     console (1.16.2)
       fiber-local
+    falcon (0.42.3)
+      async
+      async-container (~> 0.16.0)
+      async-http (~> 0.57)
+      async-http-cache (~> 0.4.0)
+      async-io (~> 1.22)
+      build-environment (~> 1.13)
+      bundler
+      localhost (~> 1.1)
+      openssl (~> 3.0)
+      process-metrics (~> 0.2.0)
+      protocol-rack (~> 0.1)
+      samovar (~> 2.1)
     fiber-local (1.0.0)
-    io-event (1.1.6)
+    httpray (1.1.3)
+    io-event (1.1.7)
+    localhost (1.1.10)
+    mapping (1.1.1)
+    mustermann (3.0.0)
+      ruby2_keywords (~> 0.0.1)
+    openssl (3.1.0)
+    process-metrics (0.2.1)
+      console (~> 1.8)
+      samovar (~> 2.1)
+    protocol-hpack (1.4.2)
+    protocol-http (0.24.1)
+    protocol-http1 (0.15.0)
+      protocol-http (~> 0.22)
+    protocol-http2 (0.15.1)
+      protocol-hpack (~> 1.4)
+      protocol-http (~> 0.18)
+    protocol-rack (0.2.4)
+      protocol-http (~> 0.23)
+      rack (>= 1.0)
+    rack (2.2.6.4)
+    rack-protection (3.0.5)
+      rack
+    ruby2_keywords (0.0.5)
+    samovar (2.1.4)
+      console (~> 1.0)
+      mapping (~> 1.0)
+    sinatra (3.0.5)
+      mustermann (~> 3.0)
+      rack (~> 2.2, >= 2.2.4)
+      rack-protection (= 3.0.5)
+      tilt (~> 2.0)
+    tilt (2.1.0)
     timers (4.3.5)
+    traces (0.9.1)
 
 PLATFORMS
+  arm64-darwin-22
   x86_64-linux
 
 DEPENDENCIES
   async
+  async-http
+  falcon
+  httpray
+  sinatra
+
+RUBY VERSION
+   ruby 3.2.0p0
```

Comment:
should I remove this version ?  :P  

I don't know how to bundle without it (ofc without manual edit)